src/Core/Geographic/Coordinates.js

```diff
@@ -110,26 +110,38 @@ function instanceProj4(crsIn, crsOut) {
 }
 
 // Only support explicit conversions
-function _convert(coordsIn, newCrs) {
+function _convert(coordsIn, newCrs, target) {
     if (newCrs === coordsIn.crs) {
         const refUnit = crsToUnit(newCrs);
         if (coordsIn._internalStorageUnit != refUnit) {
             // custom internal unit
             if (coordsIn._internalStorageUnit == UNIT.DEGREE && refUnit == UNIT.RADIAN) {
-                return new Coordinates(newCrs, ...coordsIn._values.map(x => mE.degToRad(x)));
+                target = target || new Coordinates(newCrs, 0, 0);
```

Comment:
move `target = target || new Coordinates(newCrs, 0, 0)` between line 113/114
to reduce code